packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/simulationservice/SimulationRequestResource.java

```diff
@@ -20,12 +22,19 @@
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
+import java.util.*;
+
+import software.uncharted.terarium.hmiserver.proxies.dataservice.ModelConfigurationProxy;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.jboss.logging.Logger;
 
 @Path("/api/simulation-request")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
 @Tag(name = "Simulation Service REST Endpoint")
 public class SimulationRequestResource {
+	private static final Logger log = Logger.getLogger(SimulationRequestResource.class);
```

Comment:
Not sure why i need to add this?

Reply:
I thought we are using that `@slf4j `stuff

Reply:
^Echoing @mwdchang here. That is *a* way to do that... but you should be able to just add with the annotation

Reply:
Thanks guys